Artifacts/linux-npm-package/Artifactfile.json

```diff
@@ -0,0 +1,36 @@
+{
+    "$schema": "https://raw.githubusercontent.com/Azure/azure-devtestlab/master/schemas/2015-01-01/dtlArtifacts.json",
+    "title": "NPM Package",
+    "description": "Install NPM packages on linux using the Node Package Manager.",
+    "tags": [
+        "NPM",
+        "Linux",
+        "Packages"
+    ],
+    "iconUri": "https://www.npmjs.com/favicon.ico",
+    "targetOsType": "Linux",
+    "parameters": {
+        "installLocation": {
+            "type": "string",
+            "displayName": "Install Location",
+            "defaultValue": "/var/tmp",
+            "allowEmpty": false,
+            "description": "The local path to install the package to or the keyword 'global' to install the package(s) globally instead."
+        },
+        "packages": {
+            "type": "string",
+            "displayName": "Packages to Install",
+            "description": "The package names you wish to have installed via NPM, separated by commas. The packages will be installed globally. If NPM is not yet on the system, it also will be installed. You can find NPM package names in the index found at https://www.npmjs.com. All npm install commands are supported, however if a git/bitbucket/gist scope is given then you must have installed the git artifact prior to this npm package artifact."
```

Comment:
Why does this say they'll be installed globally but the previous parameter provides an option to install globally or to a specific location?